src/Services/EncounterService.php

```diff
@@ -242,6 +242,54 @@ public function getEncounterForPatient($pid, $eid)
         return sqlQuery($sql, array($pid, $eid));
     }
 
+    public function insertEncounter($pid, $data)
+    {
+
+
+        $sql .= " INSERT INTO form_encounter SET";
+        $sql .= "     date =NOW(),";
```

Comment:
need to set the date rather than NOW() since this is what sets the date of the encounter

Reply:
Okay, will do